zerver/migrations/0478_add_topic_wildcard_mentioned_flag_and_create_index.py

```diff
@@ -0,0 +1,119 @@
+# Generated by Django 4.2.3 on 2023-08-16 14:16
+
+import bitfield.models
+from django.contrib.postgres.operations import AddIndexConcurrently, RemoveIndexConcurrently
+from django.db import migrations, models
+from django.db.backends.base.schema import BaseDatabaseSchemaEditor
+from django.db.migrations.state import StateApps
+from django.db.models import Q
+
+
+def clear_set_bit_topic_wildcard_and_group_mentioned(
+    apps: StateApps, schema_editor: BaseDatabaseSchemaEditor
+) -> None:
```

Comment:
Let's add a blockquote here explaining that because these flags are reused flag slots in the `flags` bitword, we're not confident that their value is always in the "fresh" always-0 state in very old servers, and this migration to ensure that's the case.

Reply:
I have added a comment in 0486 migration file.